cilium-health/launch/endpoint.go

```diff
@@ -274,14 +274,25 @@ func LaunchAsEndpoint(baseCtx context.Context,
 
 	switch option.Config.DatapathMode {
 	case datapathOption.DatapathModeVeth:
-		_, epLink, err := connector.SetupVethWithNames(vethName, epIfaceName, mtuConfig.GetDeviceMTU(),
+		_, epLink, err := connector.SetupVethWithNames(healthName, epIfaceName, mtuConfig.GetDeviceMTU(),
 			bigTCPConfig.GetGROIPv6MaxSize(), bigTCPConfig.GetGSOIPv6MaxSize(),
 			bigTCPConfig.GetGROIPv4MaxSize(), bigTCPConfig.GetGSOIPv4MaxSize(),
 			info, sysctl)
 		if err != nil {
 			return nil, fmt.Errorf("Error while creating veth: %w", err)
 		}
-
+		if err = netlink.LinkSetNsFd(epLink, int(ns.FD())); err != nil {
+			return nil, fmt.Errorf("failed to move device %q to health namespace: %w", epIfaceName, err)
+		}
+	case datapathOption.DatapathModeNetkit, datapathOption.DatapathModeNetkitL2:
+		l2Mode := option.Config.DatapathMode == datapathOption.DatapathModeNetkitL2
+		_, epLink, err := connector.SetupNetkitWithNames(healthName, epIfaceName, mtuConfig.GetDeviceMTU(),
```

Comment:
just question. What does MTU option mean in netkit context? Is it just ETH_MAX_MTU?

Reply:
ah figured it out. I thought if no ndo_change_mtu existed setting mtu would cause an issue but stack handles it fine. Nothing to worry about.

Reply:
Yeah, MTU setting is no different than veth in netkit.
> I thought if no ndo_change_mtu existed setting mtu would cause an issue but stack handles it fine.
Could you elaborate? I was thinking to extend netkit with ndo_change_mtu at some point to align MTUs for both devs when changed on primary device.